cloud/deployment/deployment.go

```diff
@@ -945,6 +947,8 @@ func Update(deploymentID, name, ws, description, deploymentName, dagDeploy, exec
 				standardDeploymentRequest.SchedulerSize = astroplatformcore.UpdateStandardDeploymentRequestSchedulerSizeMEDIUM
 			case LargeScheduler:
 				standardDeploymentRequest.SchedulerSize = astroplatformcore.UpdateStandardDeploymentRequestSchedulerSizeLARGE
+			case ExtraLargeScheduler:
+				standardDeploymentRequest.SchedulerSize = astroplatformcore.UpdateStandardDeploymentRequestSchedulerSizeEXTRALARGE
```

Comment:
Should we sprinkle a standard deployment test in to make this warning go away?

Reply:
added the unit test 🙃